### ISSUE TITLE
Opendata: Automate DCM Dataset push (6)

### ISSUE THREAD

**Issue by alexrichey** [NYCPlanning/data-engineering#795]
The [Digital City Map – Shapefile](https://data.cityofnewyork.us/City-Government/Digital-City-Map-Shapefile/m2vu-mgzw) 
Needs to be broken into 6 different Socrata pages - one for each layer in the fgdb. The metadata will need to be pulled from the existing README.

Then the [fgdb dataset](https://data.cityofnewyork.us/d/eak9-f97n) _probably_ to be retired, perhaps just replaced with a link back to Bytes.

New Datasets:
dcm: qrhk-k5wn

arterials_major_streets: 6r5z-bkpx

street_center_line: er35-3npf

street_name_changes: nzui-5vt3, i3nt-3kau, hsi9-e2ec

city_map_alterations: 5iv5-v7sg

**Comment by damonmcc** [comment]
DCM updates may be on-hold since the schema are being changed all summer

**Comment by alexrichey** [comment]
FYI, we can still get going on automating these, it's just a question of whether we publish. But they're low-priority for the next two-weeks while we focus on:
- Zoning Application Portal (ZAP)
- Mandatory Inclusionary Housing (MIH)
- Capital Projects Database (CPDB)
- E Designations (EDes)
- LION/Districts Quarterly Release

**Comment by alexrichey** [comment]
@sf-dcp  @fvankrieken @HengJiang0206  FYI, tracking spreadsheet [here](https://nyco365.sharepoint.com/:x:/r/sites/NYCPLANNING/itd/edm/_layouts/15/Doc.aspx?sourcedoc=%7B6735537B-8D8E-437A-B5E5-3A6B54194B24%7D&file=Open_Data.xlsx&action=default&mobileredirect=true)

**Comment by alexrichey** [comment]
@fvankrieken doing ZAP

**Comment by alexrichey** [comment]
@croswell81 In the tracking spreadsheet, I added a tab* called "Socrata Review" that lists all** the datasets that need review, including DCM and Zoning. Let me know when you start reviewing - happy to answer questions, and fix things as you find them.

* I copied Jack's tab for LION, because I wanted to change a few things, but didn't want to disturb his work.
** We need to tweak HousingDB and ZoningIndex, then I'll add them to that list. _Then_ it'll be fully comprehensive.
